monitor/dbt_project/macros/get_tests_sample_data.sql

```diff
@@ -35,7 +35,11 @@
         {% set status = elementary.insensitive_get_dict_value(test, 'status') | lower %}
 
         {% set test_rows_sample = none %}
-        {%- if (test_type == 'dbt_test' and status in ['fail', 'warn']) or (test_type != 'dbt_test' and status != 'error') -%}
+        {% set elementary_tests_filtered_status = ['error'] %}
```

Comment:
cosmetics - I'd be explicit here and instead of using blacklist I'd use a whitelist - 
elementary_tests_whitelist_status = ['fail', 'warn'] and if the flag is not enabled then you can add 'pass'